src/throttle.decorator.ts

```diff
@@ -1,25 +1,33 @@
 import { THROTTLER_LIMIT, THROTTLER_SKIP, THROTTLER_TTL } from './throttler.constants';
 
-function setThrottlerMetadata(target: Function, limit: number, ttl: number): void {
```

Comment:
Any reason to remove this function? Figured having it as a separate function would make it easier to modify if need be

Reply:
reverted it.